pilot/pkg/proxy/envoy/v2/lds_test.go

```diff
@@ -239,23 +239,23 @@ func TestLDSWithDefaultSidecar(t *testing.T) {
 		return
 	}
 
-	// Expect 6 listeners : 2 orig_dst, 1 http inbound + 4 outbound (http, tcp1, istio-policy and istio-telemetry)
-	// plus 2 extra due to the mem registry
+	// Expect 7 listeners : 2 orig_dst, 1 http inbound + 4 outbound (http, tcp1, istio-policy and istio-telemetry)
 	if (len(adsResponse.HTTPListeners) + len(adsResponse.TCPListeners)) != 7 {
-		t.Fatalf("Expected 8 listeners, got %d\n", len(adsResponse.HTTPListeners)+len(adsResponse.TCPListeners))
+		t.Fatalf("Expected 7 listeners, got %d\n", len(adsResponse.HTTPListeners)+len(adsResponse.TCPListeners))
```

Comment:
A general problem with pilot tests - I think we can do better than just checking the length of the response. Just checking the number seems like we are just trying to get code coverage without wanting to write a real test

Reply:
Honestly I don't have the full pictures of the test. I feel it is the regression test: Better not to verify too many details at this point.